integration_test.go

```diff
@@ -649,11 +661,14 @@ func TestIntegrationClose(t *testing.T) {
 
 		testClose(t, session.Close)
 
-		err = receiver.HandleMessage(context.Background(), nil)
+		msg, err := receiver.Receive(context.Background())
 		if err != amqp.ErrSessionClosed {
```

Comment:
You didn't change this but would the correct thing be to errors.Is() here as well? Or would convention say "this is never wrapped so you can compare it directly"?

Reply:
A lot of this code predates `errors.Is()`, we should be using it.

Reply:
Errors.Is is go 1.13
go-amqp is still go 1.12